HCollections/HListT.fsi

```diff
@@ -61,3 +61,9 @@ module HListT =
 
     /// Obtain the corresponding TypeList for the heterogeneous types in an HListT.
     val toTypeList<'ts, 'elem> : HListT<'ts, 'elem> -> TypeList<'ts>
+
+    /// Given an HListT, returns the HList component
```

Comment:
This description is a little hard to parse from the perspective of someone who doesn't know the library. This is morally `HListT.map snd`. I'd phrase the comment as "Given an HListT, returns the corresponding HList of heterogeneous types, discarding the homogeneous element of each entry."

Similarly below.